### PR TITLE
Transaction details

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Make it possible to see details of a transaction in the transaction log.

## Changes
- The items in the transaction list are now clickable, and opens the transaction details view.

## Checklist
- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionElement.tsx [hunk 1]
```diff
@@ -161,15 +162,22 @@ function isGreenAmount(transaction: BrowserWalletTransaction, accountAddress: st
...
     const transactionTime = onlyTime(dateFromTimestamp(transaction.time, TimeStampUnit.seconds));
     const failed = transaction.status === TransactionStatus.Failed;
 
     return (
         <div
-            className={clsx('transaction-element', failed && 'transaction-element__failed')}
+            className={clsx(
+                'transaction-element',
+                failed && 'transaction-element__failed',
+                onClick && 'transaction-element__clickable'
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionElement.tsx:174]: :exploding_head:

Comment (orhoj) [reply packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionElement.tsx:174]: :boom:

File: packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss [hunk 2]
```diff
@@ -87,3 +94,54 @@
...
+        position: relative;
+        padding: 0 rem(16px) 0 rem(8px);
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        border-top: 0.5px solid transparent;
+        border-top-color: $color-grey;
+        height: rem(50px);
+        word-break: break-all;
+    }
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss:135]:
```suggestion
    &__item {
        height: rem(50px);
        padding-top: 0;
        padding-bottom: 0;
    }    
    &__dynamic-height-item,
    &__item {
        position: relative;
        padding-left: rem(16px);
        padding-right rem(8px);
        display: flex;
        flex-direction: column;
        justify-content: center;
        border-top: 0.5px solid transparent;
        border-top-color: $color-grey;
        word-break: break-all;
    }
```
I'm not a fan of specifying and overwritten properties like this (might be an irrational fear though).

File: packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss [hunk 3]
```diff
@@ -38,13 +38,20 @@
...
 .transaction-element {
     position: relative;
-    padding: 0 20px 0 10px;
+    padding: 0 rem(16px) 0 rem(8px);
     display: flex;
     flex-direction: column;
     justify-content: center;
-    border-top: 0.5px solid transparent;
+    border-top: rem(0.5px) solid transparent;
     height: 58px;
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss:46]:
```suggestion
    height: rem(58px);
```
I assume also?

File: packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    return (
+        <div className="transaction-details">
+            <div className="transaction-details__header">
+                {t('header')}
+                <CloseButton className="transaction-details__close" onClick={onClose} />
+            </div>
+            <TransactionElement accountAddress={accountAddress} transaction={transaction} />
+            {transaction.status === TransactionStatus.Failed && (
+                <div className="transaction-details__dynamic-height-item">
+                    <Title title="Reject reason" />
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionDetails.tsx:42]: i18n?

**Review by shjortConcordium — APPROVED** [review]

File: packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss (hunk 3 above)
Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/pages/Account/TransactionLog/TransactionLog.scss:46]: It should.

**Pull request merged**